src/embeddings.rs

```diff
@@ -126,6 +127,33 @@ impl PyEmbeddings {
         }
     }
 
+    fn embedding_with_norm<'p>(&self, word: &str) -> PyResult<Py<PyTuple>> {
+        let embeddings = self.embeddings.borrow();
+
+        use EmbeddingsWrap::*;
+        let embedding_with_norm = match &*embeddings {
+            View(e) => e.embedding_with_norm(word),
+            NonView(e) => e.embedding_with_norm(word),
+        };
+
+        match embedding_with_norm {
+            Some(embedding_with_norm) => {
+                let gil = pyo3::Python::acquire_gil();
+                let py = gil.python();
+                Ok((
+                    embedding_with_norm
+                        .embedding
+                        .into_owned()
+                        .into_pyarray(py)
+                        .to_owned(),
```

Comment:
I believe, `.to_owned()` is not necessary here because the embedding is owned already.